tools/mo/requirements_tf.txt

```diff
@@ -1,5 +1,5 @@
 -c ../constraints.txt
-tensorflow>=1.15.5,<2.17.0
+tensorflow>=1.15.5,<2.16.0
```

Comment:
I will probably revert this change. TF does not affect. And constraint on h5py looks to help resolving